### ISSUE TITLE
Add automated speedtest followups and store results

### ISSUE THREAD

**Issue by WillNilges** [nycmeshnet/nycmesh-support-bot#92]
We'd like a way to automatically follow up on completed installs. Two weeks after an install is completed, the support bot should run a speedtest on the node and ensure it's above some baseline, and store the result. If it's not above the baseline, it should alert someone. It could also do a speedtest on the upstream hubs to compare.

Additionally, when a speedtest is run, the results should be stored in a database, and every time a speedtest is run, the past results are called up to compare.

**Comment by WillNilges** [comment]
One way to figure out if there are new nodes is to query Andrew's API a few hours apart 2 weeks ago, vs now, and diff it. That's very un-ideal, but shrug.

**Comment by andybaumgar** [comment]
@WillNilges You could also diff the [existing API](https://api.nycmesh.net/v1/nodes).  

Actually this is an "everything table" so it's pretty big.  Might be easier to diff Andrew's data.

**Comment by margsuarez** [comment]
would also be good to somehow automate or require this to be run at initial completion of the install.   that first run would store the baseline data with which to compare.

**Comment by andybaumgar** [comment]
@margsuarez I agree we should store the results of the speed tests we already run.  We can take the max to figure out the best case.
